packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.template.html

```diff
@@ -127,7 +127,7 @@ <h4 data-translate="nasha_dashboard_partition_snapshots_subtitle"></h4>
             </tr>
             <!-- Custom snapshot form row -->
             <tr
-                class="oui-datagrid__row"
+                class="oui-datagrid__row form"
```

Comment:
Any reason to use the styles instead of the component directly ?

Reply:
You mean `oui-datagrid` ?
Yes there are two reasons actually :
- It does not provide a property to hide the pagination (doing this using css is pretty uggly IMO)
- The form row which is hidden by default. Instead of pushing / shifting the array with fake data and split the form across multiple `oui-datagrid-column`s, using styles allows me to simply ng-if the corresponding row